ibm/service/vpc/resource_ibm_is_volume.go

```diff
@@ -777,3 +777,51 @@ func deleteAllSnapshots(sess *vpcv1.VpcV1, id string) error {
 	}
 	return nil
 }
+
+func isWaitForVolAttInstanceAvailable(instanceC *vpcv1.VpcV1, id string, timeout time.Duration, d *schema.ResourceData) (interface{}, error) {
+	log.Printf("Waiting for instance (%s) to be available.", id)
+
+	communicator := make(chan interface{})
```

Comment:
I think this commincator is not required it was used  in is_instance to cover some spectial scenario..you can remove reference isVolAttInstanceRefreshFunc also